### PR TITLE
src/components/coordinator: add component TableFeeApproval for coordinator page

### PR DESCRIPTION
**Pull request by maceksimon**
Add table of pending entry fee payments for approval to be shown on company coordinator page.

* Define columns
* Predefine optional custom sorting function
* Add labels
* Enabled sorting by date (date timestamp) - perhaps by using timestamp in the data source
* Implemented a pre-sort to display "team headings"

### CODE REVIEW

File: src/composables/useTable.ts [hunk 1]
```diff
@@ -165,11 +171,101 @@ export const useTable = () => {
     }
   };
 
+  const sortByTeam = (
```
Comment (tmszi) [inline src/composables/useTable.ts:174]: Missing doc...

File: src/composables/useTable.ts [hunk 2]
```diff
@@ -165,11 +171,101 @@ export const useTable = () => {
...
+    sortBy: string,
+    descending: boolean,
+  ): readonly TableRow[] => {
+    const data = [...rows];
+
+    if (sortBy) {
+      data.sort((a, b) => {
+        const aVal = a[sortBy] ? a[sortBy] : '';
+        const bVal = b[sortBy] ? b[sortBy] : '';
+        if (!aVal || !bVal) return 0;
```
Comment (tmszi) [inline src/composables/useTable.ts:185]: Why not use `if else if else` condition statement or `switch` statement

```js
if (...) {
} else if (...) {
} else { 
}
```

?

File: src/composables/useTable.ts [hunk 3]
```diff
@@ -165,11 +171,101 @@ export const useTable = () => {
...
+        if (aVal < bVal) return descending ? 1 : -1;
+        if (aVal > bVal) return descending ? -1 : 1;
+        return 0;
+      });
+    }
+
+    data.sort((a, b) => {
+      if (!a.team || !b.team) return 0;
```
Comment (tmszi) [inline src/composables/useTable.ts:193]: Same as my previous comment https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/444/files#r1645663543.

File: src/composables/useTable.ts [hunk 4]
```diff
@@ -165,11 +171,101 @@ export const useTable = () => {
     }
   };
 
+  const sortByTeam = (
+    rows: readonly TableRow[],
+    sortBy: string,
+    descending: boolean,
+  ): readonly TableRow[] => {
+    const data = [...rows];
+
```
Comment (tmszi) [inline src/composables/useTable.ts:180]: Maybe you can split this func into sub functions...

File: src/composables/useTable.ts (hunk 1 above)
Comment (maceksimon) [reply src/composables/useTable.ts:183]: Added docs.

File: src/composables/useTable.ts (hunk 2 above)
Comment (maceksimon) [reply src/composables/useTable.ts:194]: Updated using the if - else if structure.

File: src/composables/useTable.ts (hunk 4 above)
Comment (maceksimon) [reply src/composables/useTable.ts:189]: Updated using the if - else if structure.

File: src/composables/useTable.ts (hunk 3 above)
Comment (maceksimon) [reply src/composables/useTable.ts:202]: Updated using the if - else if structure.

**Pull request merged**